jetty-server/src/main/java/org/eclipse/jetty/server/AsyncContentProducer.java

```diff
@@ -190,7 +191,7 @@ public boolean consumeAll()
         Throwable x = UNCONSUMED_CONTENT_EXCEPTION;
         if (LOG.isDebugEnabled())
         {
-            x = new IOException("Unconsumed content");
+            x = new StaticException("Unconsumed content");
```

Comment:
If DEBUG is enabled, then here we want a normal exception like it was before, with stack trace and all, no?
Also, I think this should be moved to TRACE level, as with DEBUG it is just a lot of noise in the logs in case of the happy path.